src/EditorFeatures/Test2/Diagnostics/DiagnosticServiceTests.vb

```diff
@@ -445,6 +446,7 @@ Namespace Microsoft.CodeAnalysis.Editor.Implementation.Diagnostics.UnitTests
                 Assert.Equal(0, diagnostics.Count())
 
                 diagnostics = exceptionDiagnosticsSource.TestOnly_GetReportedDiagnostics(analyzer)
+                ' This fails in V2 because exceptionDiagnosticsSource is not updated. Why?
```

Comment:
Thanks. I added the plumbing as suggested, and the test now passes. This did involve some public API changes, so I would like to get this merged for RTM. (Otherwise, we'll need to add some overloads instead of changing a couple of signatures.)